clang/cmake/caches/Peano-AIE-runtime-libraries.cmake

```diff
@@ -25,6 +25,10 @@ foreach(target ${LLVM_BUILTIN_TARGETS})
   set(BUILTINS_${target}_CMAKE_C_FLAGS "--target=${target}" CACHE STRING "")
   set(BUILTINS_${target}_CMAKE_CXX_FLAGS "--target=${target}" CACHE STRING "")
   set(BUILTINS_${target}_CMAKE_ASM_FLAGS "--target=${target}" CACHE STRING "")
+  # Pass -nostdlib to linker flags. The standard libraries don't exist yet,
+  # prevent linking them
```

Comment:
I would like to understand from this comment why someone would try to link at all when building libraries. I *think* that is because of a misconstrued use of LLVM_ENABLE_LLD. To me it makes sense to possibly use it for building library generation tooling, but then it should be referring to the host linker, not the cross linker.
Therefore, mention this CMake quirk in the comment?

Reply:
I mentioned it in the commit message. Would you prefer to have that explanation in the code?

Reply:
I think I would. As I said, this rather terse comment raises more questions than it answers.